examples/04-theming/06-code-block/App.tsx

```diff
@@ -0,0 +1,73 @@
+import "@blocknote/core/fonts/inter.css";
+import { BlockNoteView } from "@blocknote/mantine";
+import "@blocknote/mantine/style.css";
+import { useCreateBlockNote } from "@blocknote/react";
+// This packages some of the most used languages in on-demand bundle
+import { codeBlock } from "@blocknote/code-block";
+
+export default function App() {
+  // Creates a new editor instance.
+  const editor = useCreateBlockNote({
+    codeBlock,
+    initialContent: [
```

Comment:
Just a minor thing - normally in the demos we pass `PartialBlock[]` to `initialContent` rather than `Block[]`. This is to make it a bit easier to read, since you don't need to specify the ID or children, and can set the content to a string.